src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -457,6 +461,16 @@ private static void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> cal
         w.WriteLine("};");
     }
 
+    private Dictionary<Assembly, bool> assemblyDisableRuntimeMarshallingAttributeCache = new Dictionary<Assembly, bool>();
+
+    private bool HasAssemblyDisableRuntimeMarshallingAttribute(Assembly assembly)
+    {
+        if (!assemblyDisableRuntimeMarshallingAttributeCache.TryGetValue(assembly, out var value))
+            assemblyDisableRuntimeMarshallingAttributeCache[assembly] = value = assembly.GetCustomAttributesData().Any(d => d.AttributeType.Name == "DisableRuntimeMarshallingAttribute");
```

Comment:
```suggestion
            assemblyDisableRuntimeMarshallingAttributeCache[assembly]
            		= value = assembly.GetCustomAttributesData()
            								.Any(d => d.AttributeType.Name == nameof(DisableRuntimeMarshallingAttribute));
```

nit: Change the indentation here as you want, just preferably with shorter lines.

Reply:
oops, I added tabs in that.